### PR TITLE
Create script to audit AWS user accounts

### PR DESCRIPTION
**Pull request by stvstnfrd**
```sh
mkvirtualenv configuration
workon configuration
pip install boto3
./stanford/bin/audit-iam-users
```

### CODE REVIEW

**Comment by stvstnfrd** [comment]
CC: @tanyaxp

File: stanford/bin/audit-iam-users [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+
+def print_items(users):
+    for user in users:
+        print('- ' + user['user'])
+
+
+def main():
+    print('# User credential report')
+    print('')
```
Comment (caesar2164) [inline stanford/bin/audit-iam-users]: Is there really no more elegant way to add blank lines?

File: stanford/bin/audit-iam-users [hunk 2]
```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# pylint: disable=invalid-name
+# pylint: disable=missing-docstring
+from __future__ import print_function
```
Comment (caesar2164) [inline stanford/bin/audit-iam-users]: This is to use Python3's print function, right?

File: stanford/bin/audit-iam-users (hunk 1 above)
Comment (stvstnfrd) [reply stanford/bin/audit-iam-users]: Nope. Though I don't find this particularly inelegant;
it's printing nothing.

File: stanford/bin/audit-iam-users (hunk 2 above)
Comment (stvstnfrd) [reply stanford/bin/audit-iam-users]: https://docs.python.org/2/library/__future__.html
https://www.python.org/dev/peps/pep-3105/

**Pull request merged**